styles.css

```diff
@@ -142,16 +144,15 @@ footer > div > ul > li { margin-right: 21px; }
   gap: 12px;
 }
 
-.grid2 > h1 { padding-top: 12px; }
-.grid2 > p { padding-bottom: 12px; }
-.grid2 > button { margin-top: 3px; }
+.grid2 > div > h1 { padding-top: 12px; }
+.grid2 > div > p { padding-bottom: 12px; }
 
 .skills > li > ul > li {
   text-align: center;
   list-style: none;
   padding-bottom: 32px;
 }
-.grid2 > .techList > li { margin-right: 12px; }
+.grid2 > div > .techList > li { margin-right: 12px; }
```

Comment:
- [ ] Kindly fix this over widht on the tech list of your works section to fit inside the card as the FIGMA design

![image](https://user-images.githubusercontent.com/93630700/206733552-ec62d763-1cdc-41c3-b4ec-1fb15dc1dd9c.png)

Reply:
Can you please specify at what width this is happening because it looks good on my end

Reply:
![Screenshot from 2022-12-10 00-23-40](https://user-images.githubusercontent.com/107998561/206811246-22d1e0e6-9295-419f-b9a6-1cf25546986a.png)

Reply:
Hi Lorenzo, you can check it using the panel of developers on your browser.

![image](https://user-images.githubusercontent.com/93630700/206811552-9a35d004-d103-4fb6-81d8-67a047e692e0.png)